Packs/BluelivThreatContext/Integrations/BluelivThreatContext/BluelivThreatContext.py

```diff
@@ -0,0 +1,1062 @@
+        signatureIds = ""
+        if signatures > 0:
+            signatureIds = client.get_relationships("cve", str(vulnId), "signature")
+
+        # TAGS #
+        tagIds = ""
+        tags = result['data']['relationships']['tags']['meta']['count']
+        if tags > 0:
+            tagIds = client.get_relationships("cve", str(vulnId), "tag")
+
+        # CRIME SERVERS #
+        crimeServerIds = ""
+        crimeServers = result['data']['relationships']['crime_servers']['meta']['count']
+        if crimeServers > 0:
+            crimeServerIds = client.get_relationships("cve", str(vulnId), "crime-server")
+
+        # SPARKS #
+        sparkIds = ""
+        sparks = result['data']['relationships']['sparks']['meta']['count']
+        if sparks > 0:
+            sparkIds = client.get_relationships("cve", vulnId, "spark")
+
+        # MALWARE #
+        malwareIds = ""
+        malware = result['data']['relationships']['malware']['meta']['count']
+        if malware > 0:
+            malwareIds = client.get_relationships("cve", vulnId, "malware")
+
+        human = getHuman(result)
+
+        human = getHuman(result)
+        demisto.results({
+            "Type": entryTypes["note"],
+            'Contents': result,
+            "ContentsFormat": formats["json"],
+            'HumanReadable': tableToMarkdown("Blueliv CVE info", human),
+            'ReadableContentsFormat': formats['markdown'],
+            'EntryContext': {
+                'threatContext.hasResults': True,
+                'cve.name': name,
+                'cve.description': description,
+                'cve.updatedAt': updatedAt,
+                'cve.score': score,
+                'cve.attackPatterns': attackPatterns,
+                'cve.attackPatternIds': attackPatternIds,
+                'cve.signatures': signatures,
+                'cve.signatureIds': signatureIds,
+                'cve.tags': tags,
+                'cve.tagIds': tagIds,
+                'cve.crimeServers': crimeServers,
+                'cve.crimeServerIds,': crimeServerIds,
+                'cve.sparks': sparks,
+                'cve.sparkIds': sparkIds,
+                'cve.malware': malware,
+                'cve.malwareIds': malwareIds,
+                'cve.exploits': exploitsTableData,
+                'cve.platforms': platformsTableData
+            }
+        })
+    else:
+        notFound()
+
+
+# DEMISTO command evaluation
+def main():
+    params = demisto.params()
+    server_url = params.get('url')
+    verify_ssl = not params.get('unsecure', False)
+    proxy = params.get('proxy')
+    username = params['credentials']['identifier']
+    password = params['credentials']['password']
+
+    client = Client(server_url, verify_ssl, proxy, headers={'Accept': 'application/json'})
+    token = client.authenticate(username, password)
+
+    args = demisto.args()
+    if demisto.command() == 'test-module':
+        demisto.results("ok")
```

Comment:
add a comment here that indicates the test-module checks for successful authentication